src/forms/TogglePill.jsx

```diff
@@ -22,7 +22,6 @@ class TogglePill extends React.PureComponent {
 			topic,
 			id,
 			small,
-			large,
```

Comment:
You may want to leave this here, or `delete other.large` to pluck the prop from the spread in case any consumers still pass the `large` prop.